storage/remote/queue_manager.go

```diff
@@ -544,8 +549,9 @@ func (t *QueueManager) calculateDesiredShards() int {
 
 	// When behind we will try to catch up on a proporation of samples per tick.
 	// This works similarly to an integral accumulator in that pending samples
-	// is the result of the error integral.
-	const integralGain = 0.2 / float64(shardUpdateDuration/time.Second)
+	// is the result of the error integral. This effectively means try to catch
+	// up on all pending samples within the next 10 cycles (100 seconds).
```

Comment:
when the default `shardUpdateDuration` of 10s is used, we should mention that just incase the value is ever changed, or just say `within the next <math here> cycles.`